test/parallel/test-stringbytes-external.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+// Flags: --expose_internals
```

Comment:
As an idea: we can make a separate folder for tests that require `expose_internals` and set it implicitly.